src/main/java/picard/sam/markduplicates/MarkDuplicates.java

```diff
@@ -182,6 +184,9 @@
     @Argument(doc= "Clear DT tag from input SAM records. Should be set to false if input SAM doesn't have this tag.  Default true")
     public boolean CLEAR_DT = true;
 
+    @Argument(doc= "Treat UMIs as duplex strand.  If true, UMIs will be accessed through the tag specified by the BARCODE_TAG argument.  Default false.")
```

Comment:
explain more precisely what you mean by "duplex strand" and what the effect of this will be.